code/controllers/configuration/entries/motd.dm

```diff
@@ -0,0 +1 @@
+/datum/config_entry/str_list/motd
```

Comment:
Why put this in it's own file and not just general.dm?

Reply:
No real reason other than I dislike huge miscellaneous files, and files are free.

I can put it in if you want.

Reply:
I'd normally agree with you, but since this is literally a one line definition, I think grouping them is better.